src/renderer/components/input/input.scss

```diff
@@ -63,6 +63,10 @@
     }
   }
 
+  input[type=number]::-webkit-inner-spin-button {
+    -webkit-appearance: none; // hide browser controls (top/bottom arrows)
+  }
```

Comment:
do we really need to hide these?

Reply:
<img width="609" alt="Screenshot 2021-11-03 at 11 28 33" src="https://user-images.githubusercontent.com/6377066/140036491-abf58125-535b-4036-99e4-eb94bf91d708.png">

Reply:
ok